### PR TITLE
Assets: protocol rebranding

### PR DESCRIPTION
**Pull request by rperez89**
This Pr includes all the new illustrations for the aragon protocol and some changes to fixed images sizes

### CODE REVIEW

File: src/assets/emailNotifications.svg [hunk 1]
```diff
@@ -1 +1,44 @@
...
\ No newline at end of file
+<svg width="205" height="173" viewBox="0 0 205 173" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (fabriziovigevani) [inline src/assets/emailNotifications.svg:1]: Lets optimize this one

File: src/assets/noResultsSmall.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...9 10.564 38.253 0 10.564-10.563 10.564-27.69 0-38.253-10.563-10.564-27.69-10.564-38.253 0-10.564 10.563-10.564 27.69 0 38.254z" fill="#B6D5E1"/><path d="M76.131 62.368V58.28l2.245-2.041 5.473 4.502-3.859 5.211-3.859-3.585zM123.811 54.504l-22.995 13.184-3.758-.273-2.85 2.176 3.304 8 6.123-3.21 18.537-8.699a5.258 5.258 0 002.881-5.984l-1.242-5.194z" fill="#F5C5B6"/><path d="M112.323 48.385l-.451.135 3.251 10.91.451-.134-3.251-10.91zM86.42 51.776l-.459.098 1.987 9.288.46-.098-1.987-9.288z" fill="#203254"/><path d="M98.025 34.03c.825-1.347 1.92-1.988 3.411-1.988" stroke="#203254"/><defs><linearGradient id="paint0_linear" x1="32.268" y1="102.678" x2="73.665" y2="35.177" gradientUnits="userSpaceOnUse"><stop stop-color="#FFBE81"/><stop offset="1" stop-color="#FF5F5F"/></linearGradient><linearGradient id="paint1_linear" x1="-13.323" y1="91.442" x2="58.97" y2=".773" gradientUnits="userSpaceOnUse"><stop stop-color="#FFD3AA"/><stop offset="1" stop-color="#FF9696"/></linearGradient><linearGradient id="paint2_linear" x1="-159.385" y1="232.647" x2="-92.985" y2="-8.881" gradientUnits="userSpaceOnUse"><stop stop-color="#FFBE81"/><stop offset="1" stop-color="#FF5F5F"/></linearGradient></defs></svg>
```
Comment (fabriziovigevani) [inline src/assets/noResultsSmall.svg:1]: Why not use the original asset and set the height and with to the `img` ?

File: src/components/Disputes/NoEvidence.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 import MessageCard from '../MessageCard'
-import noEvidenceSvg from '../../assets/noEvidence.svg'
+import noEvidenceSvg from '../../assets/noDraft.svg'
```
Comment (fabriziovigevani) [inline src/components/Disputes/NoEvidence.js:4]: We might want to rename `noDraft` to something more general now.

**Review by fabriziovigevani — APPROVED** [review]
Looks good! 

Couple of comments.

**Comment by sohkai** [comment]
Should we close this for https://github.com/aragon/protocol-dashboard/pull/1?

**Comment by rperez89** [comment]
> Should we close this for [aragon/protocol-dashboard#1](https://github.com/aragon/protocol-dashboard/pull/1)?

Yes! i already moved the prs to the protocol dashboard with the comments done

**Pull request closed without merging**